docs/api-references/user.md

```diff
@@ -0,0 +1,123 @@
+## User API
+
+### `GET` User
+```
+/api/users/
```

Comment:
To display information of current user login, you can use endpoint `/api/users/me/`